index.js

```diff
@@ -0,0 +1,147 @@
+const day = document.getElementById("js-day"),
+ date = document.getElementById("js-date"),
+ month = document.querySelector(".js-month");
+
+ today = new Date(),
+ MONTH = today.getMonth(),
+ YEAR = today.getFullYear(),
+ PAINTING = "painting";
+
+// 현재 요일, 일자
+function currentDay() {
+    const week = new Array('SUN', 'MON', 'TUE', 'WED', 'THU', 'FRI', 'SAT');
```

Comment:
`new` 키워드를 이용해서 배열을 생성하셨는데, 배열이나 객체는 literal 방식으로 선언하는 것을 권장합니다.
```suggestion
    const week = ['SUN', 'MON', 'TUE', 'WED', 'THU', 'FRI', 'SAT'];
```